dnf5/main.cpp

```diff
@@ -267,10 +267,17 @@ void RootCommand::set_argument_parser() {
     auto skip_broken = parser.add_new_named_arg("skip-broken");
     skip_broken->set_long_name("skip-broken");
     skip_broken->set_description("resolve depsolve problems by skipping packages");
-    skip_broken->set_const_value("false");
-    skip_broken->link_value(&config.strict());
+    skip_broken->set_const_value("true");
+    skip_broken->link_value(&config.skip_broken());
```

Comment:
I think that description of both option is on incorrect place. Those options are applicable only for a set of transaction commands - for upgrade does not make sense to allow skip-broken and so on. What about to use a similar approach like we have for security options `--security`

Reply:
That's a good point. I wasn't completely sure about making them global options. I'll move them to specific commands.

Reply:
The change (move from global options to specific commands) was a bit bigger than expected. I added it as 3 new commits to this PR - basically to make reviewer's life easier (and my also :)

Reply:
Feel free to create an issue for dnf5 project. I am fine with not implementing what I suggested in this PR.